tests/test_pod.py

```diff
@@ -5,23 +5,44 @@
 
 snapshots = np.load('tests/test_datasets/p_snapshots.npy').T
 poddb = np.load('tests/test_datasets/p_snapshots_pod.npy')
+modes = np.load('tests/test_datasets/p_snapshots_pod_modes.npy')
 
 
 class TestPOD(TestCase):
     def test_constructor_empty(self):
         a = POD()
 
-    def test_1(self):
+    def test_numpysvd(self):
         A = POD('svd').reduce(snapshots)
         assert np.allclose(A, poddb, rtol=1e-03, atol=1e-08) or np.allclose(
             A, -1 * poddb, rtol=1e-03, atol=1e-08,)
 
-    def test_2(self):
+    def test_correlation_matirix(self):
         A = POD('correlation_matrix').reduce(snapshots)
         assert np.allclose(A, poddb, rtol=1e-03, atol=1e-08) or np.allclose(
             A, -1 * poddb, rtol=1e-03, atol=1e-08,)
-
-    def test_3(self):
-        A = POD('randomized_svd').reduce(snapshots)
+    
+    def test_correlation_matirix_savemem(self):
+        A = POD('correlation_matrix',save_memory = True).reduce(snapshots)
```

Comment:
please add a space after the comma, and remove the spaces before/after the equal sign